.github/ISSUE_TEMPLATE/FEATURE-REQUEST.yml

```diff
@@ -0,0 +1,47 @@
+name: Feature Request
+description: 🚨 Please make sure to fill out the template carefully. Otherwise, we may not be able to assist you. 🚨
+title: "[Feature]: "
+labels: ["✨ feature", "📋 needs review"]
+assignees:
+  - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to fill out this feature request. 🚀
```

Comment:
Suggestion: "If you want to report a bug, please use the bug report form (link)."

Reply:
Ok